pages/index.js

```diff
@@ -66,11 +66,19 @@ export default function Home({
               <p className="pt-4 font-bold pb-8">
                 September 22nd–25th • Miami Beach, Florida
               </p>
-              <Link href="https://assembly.urbit.org" passHref>
-                <a className="button-lg max-w-fit bg-green-400 text-white">
-                  Learn More
-                </a>
-              </Link>
+
+              <div class="flex flex-wrap">
+                <Link href="https://assembly.urbit.org" passHref>
+                  <a className="button-lg max-w-fit bg-wall-600 text-white mr-3 mb-8">
+                    Learn More
+                  </a>
+                </Link>
+                <Link href="/groups/~bitdeg/assembly22" passHref>
+                  <a className="button-lg max-w-fit bg-transparent text-wall-600 border-2 border-wall-600">
```

Comment:
Maybe `hover:bg-wall-600 hover:bg-transparent`? Looks weird as is imo but up to you.